ggshield/cmd/secret/scan/precommit.py

```diff
@@ -37,8 +38,25 @@ def precommit_cmd(
             ignored_detectors=config.secret.ignored_detectors,
         )
 
-        return output_handler.process_scan(
+        return_code = output_handler.process_scan(
             ScanCollection(id="cached", type="pre-commit", results=results)
         )
+        if return_code:
+            remediation_steps = """  Since the secret was detected before the commit was made:
+  1. replace the secret with its reference (e.g. environment variable).
+  2. commit again.
+"""
+            bypass_message = """  - if you use the pre-commit framework:
+      SKIP=ggshield git commit -m "<your message>"
+  - otherwise (warning: the following command bypasses all pre-commit hooks):
+      git commit -m "<your message>" --no-verify
+"""
```

Comment:
I am not sure about module level, since it will be used only at this part of the code. I could do this way maybe?
```
remediation_message(
                    remediation_steps="""  Since the secret was detected before the commit was made:
  1. replace the secret with its reference (e.g. environment variable).
  2. commit again.""",
                    bypass_message="""  - if you use the pre-commit framework:
      SKIP=ggshield git commit -m "<your message>"
  - otherwise (warning: the following command bypasses all pre-commit hooks):
      git commit -m "<your message>" --no-verify
""",
                    rewrite_git_history=False,
                ),
```

Reply:
Using them directly looks worse I think: the first line is even more indented.

Even if it's used in only one place, I think it's OK to create a constant. That helps grouping all messages at the top instead of having them buried in the code. It also makes the code easier to read, IMO.

Having said that, it's not a blocker for me. You can leave it as is if you prefer it this way.